infra/config-store/src/main/java/com/simprints/infra/config/store/remote/models/ApiFingerprintConfiguration.kt

```diff
@@ -40,6 +41,7 @@ internal data class ApiFingerprintConfiguration(
             vero2 = vero2?.toDomain(),
             allowedAgeRange = allowedAgeRange?.toDomain() ?: AgeGroup(0, null),
             verificationMatchThreshold = verificationMatchThreshold,
+            version = version.orEmpty(),
```

Comment:
The backend is not sending the version, so why add it into the `ApiFingerprintConfiguration` class?

Reply:
For consistency and to ensure that it is handled in the same way in future.

Reply:
IMO, for consistency, you should only add it in your domain class to make the impression to future readers that BFSID is sending this field

Reply:
I will remove it for now, but I still don't think it is an issue since it is defined as optional with the default set to null, which should be a clear indicator that the version is not guaranteed in the API.